rhobs/alerting/data_plane/prometheus.gitops_deployment_alerts.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-gitops-deployment-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: gitops_deployment_alerts
+    interval: 1m
+    rules:
+    - alert: GitOpsDeploymentHealthErrors
+      expr: |
+        (
+          sum(kube_deployment_status_replicas_ready{namespace="gitops-service-argocd"})
```

Comment:
Does the `kube_deployment_status_replicas_ready` metric gives the number of replicas in ready state or it gives `1` when it's ready?
What does it return when no replicas are in ready state? 0? no data?

Reply:
Row will be there but `value` will be 0, PTAL at this screenshot where 1st deployment is not in healthy.

![image](https://github.com/redhat-appstudio/o11y/assets/63054462/e4aae8a9-4d6b-4ee4-a611-0690d10cf7d0)